app/config/mq-config.js

```diff
@@ -1,33 +1,39 @@
-const joi = require('joi')
+const Joi = require('joi')
 
-const mqSchema = joi.object({
+const mqSchema = Joi.object({
   messageQueue: {
-    host: joi.string(),
-    username: joi.string(),
-    password: joi.string(),
-    useCredentialChain: joi.bool().default(false),
-    type: joi.string().default('subscription'),
-    appInsights: joi.object()
+    host: Joi.string().required(),
+    username: Joi.string(),
+    password: Joi.string(),
+    useCredentialChain: Joi.bool().default(false),
+    appInsights: Joi.object()
   },
   processingSubscription: {
-    name: joi.string(),
-    address: joi.string(),
-    topic: joi.string(),
-    numberOfReceivers: joi.number().default(3)
+    address: Joi.string().required(),
```

Comment:
Adding `required` to ensure names are captured.  Joi behaviour seems to have changed around required strings, so adding to ensure mandatory data provided.

Also dropping `name` value as it is not required in `ffc-messaging` and isn't used anywhere in this repo.